src/typography.js

```diff
@@ -18,7 +18,7 @@ const Typography = forwardRef(({ marginBottom, children, secondary, ...rest }, r
 
 Typography.propTypes = { 
   marginBottom: PropTypes.number,
-  children: PropTypes.string.isRequired,
+  children: PropTypes.node.isRequired,
```

Comment:
would this cover string as well or should we use the "one of" interface

Reply:
From what I read here: https://developer.mozilla.org/en-US/docs/Web/API/Node/nodeType type node includes text. Also, tested locally in poi-manage and I don't get any warning anymore. The red box includes 2 Typography elements, first one text type, second (with italic _about_) is a div elem.
<img width="1577" alt="Screen Shot 2021-11-03 at 12 49 57 PM" src="https://user-images.githubusercontent.com/41120953/140108727-9491893c-e4d1-4cfe-8b27-3898f15dee45.png">

Reply:
i see, seems React prop-types honor that too:

> // Anything that can be rendered: numbers, strings, elements or an array
> // (or fragment) containing these types.